unit_tests/provisioner/test_provision_sct_resources.py

```diff
@@ -60,9 +61,10 @@ def test_fallback_on_demand_when_spot_fails(fallback_on_demand, params, test_con
     fake_remoter.result_map = {r"sudo cloud-init status --wait": Result(stdout="..... \n status: done", stderr="nic", exited=0),
                                r"sudo cloud-init --version": Result(stdout="/bin/ 19.2-amzn", stderr="", exited=0),
                                r"ls /var/lib/sct/cloud-init": Result(stdout="done", exited=0)}
-    provision_sct_resources(params=params, test_config=test_config, azure_service=azure_service)
+    provision_sct_resources(params=params, test_config=test_config,
+                            azure_service=azure_service, enterprise_disable_kms=True)
     provisioner_eastus = provisioner_factory.create_provisioner(
-        backend="azure", test_id=params.get("test_id"), region="eastus", availability_zone="1", azure_service=azure_service)
+        backend="azure", test_id=params.get("test_id"), region="eastus", availability_zone="1", azure_service=azure_service, params=params, enterprise_disable_kms=True)
```

Comment:
Why the `enterprise_disable_kms=True` is provided explicitly?
Why not taken from params further as it is in the main functionality?